feature-libs/product-configurator/rulebased/components/attribute/types/multi-selection-image/configurator-attribute-multi-selection-image.component.ts

```diff
@@ -46,6 +48,11 @@ export class ConfiguratorAttributeMultiSelectionImageComponent
     this.attribute = attributeComponentContext.attribute;
     this.ownerKey = attributeComponentContext.owner.key;
     this.expMode = attributeComponentContext.expMode;
+    this.initDeltaRendering(
+      attributeComponentContext.isDeltaRendering,
+      attributeComponentContext.attribute.key
+    );
```

Comment:
Do we _really_ need to pass `isDeltaRendering` as a method parameter?
Can't the super class simply inject the `FeatureConfigService` itself and check if the toggle is enabled?